primeqa/ir/dense/colbert_top/colbert/search/index_storage.py

```diff
@@ -134,7 +134,7 @@ def score_pids(self, config, Q, pids, centroid_scores):
                 codes_packed_ = codes_packed[idx_]
                 approx_scores_ = centroid_scores[codes_packed_.long()]
                 if approx_scores_.shape[0] == 0:
-                    approx_scores.append(torch.zeros((len(pids_),), dtype=approx_scores_.dtype))
```

Comment:
shouldn't this be wrapped in `if self.use_gpu:`?

Reply:
there are other examples of this in the same file

Reply:
> shouldn't this be wrapped in `if self.use_gpu:`?

Isn't it already (line 123)?

Reply:
> there are other examples of this in the same file

Please say more, where in the file?